src/edges_analysis/analysis/averaging.py

```diff
@@ -326,7 +326,7 @@ def bin_gha_unbiased_regular(
     weights: np.ndarray,
     gha: np.ndarray,
     bins: np.ndarray,
-) -> [np.ndarray, np.ndarray, np.ndarray]:
+) -> Tuple[np.ndarray, np.ndarray, np.ndarray]:
```

Comment:
How did the capital 'T' not throw an error?

Reply:
@steven-murray

Reply:
this is the "typing annotation" Tuple, not the actual "tuple". It's meant to say it will have the type of tuple, without actually using a tuple itself to describe it

Reply:
I see. Definitely did not know that. :)